### PR TITLE
[DOCS-7209] updating network vendors and SD-WAN support

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do? What is the motivation?
This PR adds a table for the full list of supported network vendors and links to their integration (where applicable), as well as links to their config (yaml) files for the network profile.
Also adds support for SD-WAN devices (beta)

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/network_monitoring/devices/_index.md [hunk 1]
```diff
@@ -56,35 +55,96 @@ Network Device Monitoring gives you visibility into your on-premise and virtual
...
+| Eaton | [eaton-epdu.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/eaton-epdu.yaml) <br> |
+| Exagrid | [exagrid.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/exagrid.yaml) <br> |
+| Extreme Networks | [extreme-switching.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/extreme-switching.yaml) <br> |
+| [F5](/integrations/snmp_f5/) | [f5-big-ip.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/f5-big-ip.yaml) <br> |
+| Fireeye | [fireeye.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/fireeye.yaml) <br> |
+| [Fortinet](integrations/snmp_fortinet/) | [fortinet.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/fortinet.yaml) <br>[fortinet-fortigate.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/fortinet-fortigate.yaml) <br> |
```
Comment (buraizu) [inline content/en/network_monitoring/devices/_index.md:104]:
```suggestion
| [Fortinet](https://docs.datadoghq.com/integrations/snmp_fortinet/) | [fortinet.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/fortinet.yaml) <br>[fortinet-fortigate.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/fortinet-fortigate.yaml) <br> |
```

File: content/en/network_monitoring/devices/_index.md [hunk 2]
```diff
@@ -56,35 +55,96 @@ Network Device Monitoring gives you visibility into your on-premise and virtual
...
+| Huawei | [_huawei.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/_huawei.yaml) <br> |
+| iXsystems | [ixsystems-truenas.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/ixsystems-truenas.yaml) <br> |
+| IBM | [ibm.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/ibm.yaml) <br> |
+| Infinera | [infinera-coriant-groove.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/infinera-coriant-groove.yaml) <br> |
+| [Infoblox](integrations/rapdev_infoblox/) | [infoblox-ipam.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/infoblox-ipam.yaml) <br> |
```
Comment (buraizu) [inline content/en/network_monitoring/devices/_index.md:111]:
```suggestion
| [Infoblox](https://docs.datadoghq.com/integrations/rapdev_infoblox/) | [infoblox-ipam.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/infoblox-ipam.yaml) <br> |
```

File: content/en/network_monitoring/devices/_index.md [hunk 3]
```diff
@@ -56,35 +55,96 @@ Network Device Monitoring gives you visibility into your on-premise and virtual
...
+| NetApp | [netapp.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/netapp.yaml) <br> |
+| Netgear | [netgear.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/netgear.yaml) <br> |
+| Nvidia | [nvidia.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/nvidia.yaml) <br> |
+| Omron | [omron-cj-ethernet-ip.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/omron-cj-ethernet-ip.yaml) <br> |
+| Opengear | [_opengear.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/_opengear.yaml) <br> |
+| [Palo Alto Networks](integrations/pan_firewall/) | [palo-alto.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/palo-alto.yaml) <br>[_palo-alto.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/_palo-alto.yaml) <br> |
```
Comment (buraizu) [inline content/en/network_monitoring/devices/_index.md:125]:
```suggestion
| [Palo Alto Networks](https://docs.datadoghq.com/integrations/pan_firewall/) | [palo-alto.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/palo-alto.yaml) <br>[_palo-alto.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/_palo-alto.yaml) <br> |
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks, just requesting some minor edits, as well as updating a few relative links that aren't opening the expected pages.

In addition, could you please update the new links to use the **reference** format as mentioned in the [documentation style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#text)? This is intended to aid with the translation process.

**Comment by aliciascott** [comment]
Thanks @buraizu ! I committed the changes you requested, and requested re-review. let me know if I did the links correctly, thanks for catching those.

File: content/en/network_monitoring/devices/_index.md [hunk 4]
```diff
@@ -56,35 +55,98 @@ Network Device Monitoring gives you visibility into your on-premise and virtual
...
--   Palo Alto
+The following vendor devices are supported with dedicated profiles. If a vendor or device type is supported, but the specific model isn't supported, you can:
+  - customize your own YAML profile (see the [NDM troubleshooting FAQ page][6] for guidance)
+  - create new [profiles][9] to start monitoring new device models
+
+| Vendor | Config files |
+| ---  | ----------- |
+| 3com | [3com.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/3com.yaml) <br> |
+| TP-Link | [tp-link.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/tp-link.yaml) <br> |
+| A10 | [a10.yaml](https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/a10.yaml) <br> |
```
Comment (buraizu) [inline content/en/network_monitoring/devices/_index.md:77]:
```suggestion
| 3com | [3com.yaml][11] <br> |
| TP-Link | [tp-link.yaml][12] <br> |
| A10 | [a10.yaml][13] <br> |
```

File: content/en/network_monitoring/devices/_index.md [hunk 5]
```diff
@@ -96,4 +158,7 @@ The following vendor devices are supported with dedicated profiles. If a vendor/
 [4]: https://app.datadoghq.com/dash/integration/30417/interface-performance
 [5]: /monitors/types/metric/
 [6]: /network_monitoring/devices/troubleshooting#what-do-i-do-if-datadog-supports-a-vendor-or-device-type-but-my-specific-model-isnt-supported
-[7]: https://app.datadoghq.com/devices
\ No newline at end of file
+[7]: https://app.datadoghq.com/devices
+[8]: https://github.com/DataDog/integrations-core/tree/master/snmp/datadog_checks/snmp/data/default_profiles
+[9]: /network_monitoring/devices/profiles/
+[10]: /integrations/meraki/
```
Comment (buraizu) [inline content/en/network_monitoring/devices/_index.md:164]:
```suggestion
[10]: /integrations/meraki/
[11]: https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/3com.yaml
[12]: https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/tp-link.yaml
[13]: https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/a10.yaml
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks @aliciascott for following up and making those changes. The broken links now seem to be working.

With that said, I've left some additional feedback on this review showing what I mean about the reference links. Would you be able to apply the same type of update to the other links in the table? As mentioned, this aids the translation process, and in this case also makes the table somewhat easier to read. Let me know if you have any questions!

**Comment by aliciascott** [comment]
Hey @buraizu !! Ok last time I hope :) I think I got all the links fixed now.

File: content/en/network_monitoring/devices/_index.md [hunk 6]
```diff
@@ -44,56 +43,245 @@ Network Device Monitoring gives you visibility into your on-premise and virtual
...
+| [HP][209] | [hp-ilo4.yaml][73] <br>[_hp.yaml][74] <br>[hpe-proliant.yaml][75] <br> |
+| Huawei | [_huawei.yaml][76] <br> |
+| iXsystems | [ixsystems-truenas.yaml][77] <br> |
+| IBM | [ibm.yaml][78] <br> |
+| Infinera | [infinera-coriant-groove.yaml][79] <br> |
+| [Infoblox][213] | [infoblox-ipam.yaml][80] <br> |
+| [Juniper Networks][210] | [_juniper.yaml][81] <br>[juniper-ex.yaml][82] <br>[_juniper-junos-generic.yaml][83] <br>[juniper-mx.yaml][84]
+| Linksys | [linksys.yaml][87] <br> |
+| McAfee | [mcafee-web-gateway.yaml][88] <br> |
+| [Meraki][7] | [meraki-cloud-controller.yaml][89] <br>[meraki.yaml][89] <br> |
```
Comment (buraizu) [inline content/en/network_monitoring/devices/_index.md:117]:
```suggestion
| [Meraki][7] | [meraki-cloud-controller.yaml][89] <br>[meraki.yaml][90] <br> |
```

File: content/en/network_monitoring/devices/_index.md [hunk 7]
```diff
@@ -44,56 +43,245 @@ Network Device Monitoring gives you visibility into your on-premise and virtual
...
+[88]:https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/mcafee-web-gateway.yaml
+[89]:https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/meraki-cloud-controller.yaml
+[90]:https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/meraki.yaml
+[91]:https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/mikrotik-router.yaml
+[92]:https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/nasuni-filer.yaml
+[93]:https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/nec-univerge.yaml
+[94]:https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/netapp.yaml
+[95]:https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/netgear.yaml
+[96]:ttps://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/nvidia.yaml
```
Comment (buraizu) [inline content/en/network_monitoring/devices/_index.md:229]:
```suggestion
[96]: https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/default_profiles/nvidia.yaml
```

File: content/en/network_monitoring/devices/_index.md [hunk 8]
```diff
@@ -44,56 +43,245 @@ Network Device Monitoring gives you visibility into your on-premise and virtual
...
+| Nasuni | [nasuni-filer.yaml][92] <br> |
+| NEC | [nec-univerge.yaml][93] <br> |
+| NetApp | [netapp.yaml][94] <br> |
+| Netgear | [netgear.yaml][95] <br> |
+| Nvidia | [nvidia.yaml][96] <br> |
+| Omron | [omron-cj-ethernet-ip.yaml][97] <br> |
+| Opengear | [_opengear.yaml][98] <br> |
+| [Palo Alto Networks][222] | [palo-alto.yaml][100] <br>[_palo-alto.yaml][101] <br> |
+| Peplink | [peplink.yaml][102] <br> |
+| [pfSense][33] | [pf-sense.yaml][103] <br> |
```
Comment (buraizu) [inline content/en/network_monitoring/devices/_index.md:128]:
```suggestion
| [pfSense][212] | [pf-sense.yaml][103] <br> |
```

**Review by buraizu — APPROVED** [review]
Thanks for applying those changes @aliciascott. This review corrects a few additional broken links, but overall LGTM and approving. 

As one final request, it might be helpful for future maintainers of this doc to put the reference links in numerical order.

**Pull request merged**